src/FilePOMDPs.jl

```diff
@@ -0,0 +1,145 @@
+##### 
+struct FilePOMDP{Int64} <: POMDP{Int64, Int64, Int64} 
+    number_of_states::Int64
+    number_of_actions::Int64
+    number_of_observations::Int64
+
+    support_of_distribution::Set{Int64}
+    value_of_distribution::Vector{Float64}
+
+    discount::Float64
+
+    T::TransitionProb
+    O::ObservationProb
+    R::RewardLookUp
```

Comment:
Could T, O, and R just be sparse matrices? That would decrease the amount of code we need to maintain.